docs/classic-ui/forms.md

```diff
@@ -120,6 +120,324 @@ schema = IMyForm
 If your form is not bound to an object (such as a Dexterity object), set `ignoreContext = True`.
 
 
+(classic-ui-forms-autoform-label)=
+## Controlling form presentation
+
+Directives can be specified in the schema to control aspects of form presentation.
+
+### Changing a field's display mode
+
+A field's widget can be displayed in several "modes":
+
+* input - allows the user to enter data into the field
+* display - a read-only indication of the field's value
+* hidden - a record of the field's value that is included only in the HTML source
```

Comment:
Style guide suggestions.

- Use `-` for list items.
- Indent 4 spaces.

This is more of a definition list, than a list.

```suggestion
`hidden`
:   A record of the field's value that is included only in the HTML source
```

Would you please apply the definition list pattern to these list items?